app/errors.py

```diff
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    def __init__(self) -> None:
+        super().__init__("The visitor is not vaccinated!")
+
+
+class OutdatedVaccineError(VaccineError):
+    def __init__(self) -> None:
+        super().__init__("The vaccine is outdated!")
+
+
+class NotWearingMaskError(Exception):
+    def __init__(self) -> None:
+        super().__init__("The visitor is not wearing a mask!")

```

Comment:
You should adhere to inherited logic, Exception should take a message, which could be empty, so we can run customized errors. Right now your logic breaks that.

Reply:
removed __super__(), added __str__

Reply:
Or I missunderstood you and  should put error messages by default in init method?